packages/patternfly-4/react-core/src/components/TextInput/TextInput.js

```diff
@@ -80,6 +84,7 @@ class TextInput extends React.Component {
         type={type}
         value={value}
         aria-invalid={!isValid}
+        placeholder={placeholder}
```

Comment:
This is spread as part of ...props. no need for this change. Or PR things should just work.

Reply:
I thought it but It didn't work in my environment I don't know why. I'll try again tomorrow